dev-libs/skalibs/skalibs-2.8.1.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="General-purpose libraries from skarnet.org"
+HOMEPAGE="https://www.skarnet.org/software/skalibs/"
+SRC_URI="https://www.skarnet.org/software/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0/$(ver_cut 1-2)"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="doc ipv6 static-libs"
+
+RDEPEND=""
+DEPEND=""
+
+HTML_DOCS=( doc/ )
```

Comment:
`foo/` is UB per POSIX. Use `foo/.`

Reply:
Ahh, sorry, I missed the `.` when reading your original comment